utils/commands/webpack.py

```diff
@@ -0,0 +1,14 @@
+import signal
+import subprocess
+from flask.ext.script import Command
+
+
+class Webpack(Command):
+    """Runs Webpack module bundling and hot reloading."""
+
+    def run(self):
+        subprocess.Popen(['./node_modules/.bin/webpack-dev-server'])
```

Comment:
I don't think we want to be using the `.bin` path.  The correct script to run should be
`./node_modules/webpack-dev-server/bin/webpack-dev-server.js`